Upstream: yes
Note: patch does not support binary diffs, binary parts have been stripped

From 9363a4a1e121344ae98f22b55fb28623cf0f6038 Mon Sep 17 00:00:00 2001
From: Anders Ladegaard Marchsteiner <anders.marchsteiner@kghcustoms.com>
Date: Sun, 16 Aug 2015 20:26:50 +0200
Subject: [PATCH] Kim'ie has moved Pyonyang half an hour. Not to say several
 centuries back in time...

---
 panels/datetime/Makefile.am                |   4 ++++
 1 files changed, 4 insertions(+)

diff --git a/panels/datetime/Makefile.am b/panels/datetime/Makefile.am
index 3bf0cf0..3f284ee 100644
--- a/panels/datetime/Makefile.am
+++ b/panels/datetime/Makefile.am
@@ -93,6 +93,10 @@ dist_ui_DATA = \
 	data/timezone_-8_dim.png \
 	data/timezone_8.png \
 	data/timezone_8_dim.png \
+	data/timezone_-8.5.png \
+	data/timezone_-8.5_dim.png \
+	data/timezone_8.5.png \
+	data/timezone_8.5_dim.png \
 	data/timezone_8.75.png \
 	data/timezone_8.75_dim.png \
	data/timezone_-9.png \
